src/webview/panel-views/visjsPanelView.ts

```diff
@@ -453,4 +465,16 @@ export class VisjsPanelView implements PanelViewProxy {
 
     return edges;
   }
+
+  private export(command: string, fileNameExtension: string, writer: ObjectDiagramWriter): PanelViewCommand {
+    const { currentPanelViewInput } = this;
```

Comment:
```suggestion
    const currentPanelViewInput = this.currentPanelViewInput;
```

Why so complicated? Or just use this.currentPanelViewInput instead of assigning it to a variable.

Reply:
Done.